draft-irtf-cfrg-frost.md

```diff
@@ -900,10 +900,11 @@ The value of the contextString parameter is "FROST-ED25519-SHA512-v8".
     be implemented by multiplying the resulting point by the order of the group and
     checking that the result is the identity element.
   - SerializeScalar(s): Implemented by outputting the little-endian 32-byte encoding of
-    the Scalar value.
+    the Scalar value with the top three bits set to zero.
```

Comment:
I'm not sure if this is needed, the top three bits will always be zero due to how Scalar is defined. It doesn't hurt though, it's up to you.

Reply:
It felt like @pornin's comment suggested we mention it. I feel like it can't hurt, so I'm inclined to keep it for now.